.github/workflows/dangerjs-checks.yml

```diff
@@ -0,0 +1,26 @@
+name: DangerJS PR Checks
+
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+
+jobs:
+  danger-checks:
+    runs-on: ubuntu-latest
+    steps:
+      # Checkout the Repo
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      # Setup Repo
+      - name: Setup Repo
+        uses: ./.github/actions/setup-repo
+        with:
+          node-version: 18
+          os: ubuntu-latest
+      # Run Danger Checks
+      - name: Danger
+        run: yarn danger ci --failOnErrors
+        env:
+          DANGER_GITHUB_API_TOKEN: ${{ secrets.DANGER_GITHUB_API_TOKEN }}
```

Comment:
I _think_ if you change this to `${{ secrets.CHANGESETS_TOKEN }}`, the comment will be left by the PIE bot :).

We should probably rename that token to something else, but should do the trick!

Reply:
It's already hooked up to use the pie bot (as made a new PAT for danger).

If we want to consolidate the tokens if they have the same permissions in the future, could do that too as a bit of a tidy-up?